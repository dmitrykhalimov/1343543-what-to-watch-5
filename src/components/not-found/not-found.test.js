import React from "react";
import renderer from "react-test-renderer";
import NotFound from "./error-page";

describe(`NotFound`, () => {
  it(`Should NotFound render correctly`, () => {
    const tree = renderer
      .create(
          <NotFound/>)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});

