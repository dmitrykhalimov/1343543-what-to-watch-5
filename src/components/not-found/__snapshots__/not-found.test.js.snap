// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Should NotFound render correctly 1`] = `
<div
  className="user-page"
>
  <header
    className="page-header user-page__head"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link"
        href="/"
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <h1
      className="page-title user-page__title visually-hidden"
    >
      Error loading error
    </h1>
  </header>
  <div
    className="sign-in user-page__content"
    style={
      Object {
        "maxWidth": "650px",
      }
    }
  >
    <p
      style={
        Object {
          "fontSize": "35px",
          "lineHeight": "50px",
        }
      }
    >
      Ошибка 404. Запрашиваемая страница не найдена.
    </p>
  </div>
  <footer
    className="page-footer"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link logo__link--light"
        href="/"
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <div
      className="copyright"
    >
      <p>
        © 2020 What to watch Ltd.
      </p>
    </div>
  </footer>
</div>
`;
